packages/core/src/fhir-deduplication/resources/organization.ts

```diff
@@ -1,7 +1,15 @@
 import { Organization } from "@medplum/fhirtypes";
 import { validateNPI } from "@metriport/shared";
 import { normalizeAddress } from "../../mpi/normalize-address";
-import { DeduplicationResult, combineResources, createRef, extractNpi, fillMaps } from "../shared";
+import {
+  DeduplicationResult,
+  combineResources,
+  createRef,
+  extractNpi,
+  fillL1L2Maps,
+  createKeysArray,
+  createKeys,
+} from "../shared";
 
 export function deduplicateOrganizations(
   organizations: Organization[]
```

Comment:
This is a bit out of scope, but related.
In my understanding, we will always return one map for all resources using this approach. This means we can simplify the way we return the map at the top level function in this file:

```
    combinedResources: combineResources({
      combinedMaps: [organizationsMap],
    }),
    ```

Reply:
I removed it from the relevant files and replaced it with a simple map destructuring